Simplenote/src/main/java/com/automattic/simplenote/networking/SimpleHttp.kt

```diff
@@ -0,0 +1,50 @@
+package com.automattic.simplenote.networking
+
+import okhttp3.HttpUrl
+import okhttp3.MediaType
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import okhttp3.RequestBody
+import okhttp3.Response
+import org.json.JSONException
+import org.json.JSONObject
+import javax.inject.Inject
+
+private const val HTTP_SCHEME = "https"
+private const val BASE_URL = "app.simplenote.com"
+open class SimpleHttp @Inject constructor(val client: OkHttpClient) {
```

Comment:
Consider refactoring `AccountNetworkUtils` by maybe using this class instead of the OkHtttp service it creates. It can be handled in a future PR.

Reply:
Oh hey. I guess you had a similar idea 😅 . Ya. Definitely future PR.